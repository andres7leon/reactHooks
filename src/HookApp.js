import React from 'react'

export const HookApp = () => {
    return (
        <div>
            hla mundoooo
        </div>
    )
}
