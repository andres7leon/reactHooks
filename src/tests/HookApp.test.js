import React from 'react';
import {shallow} from 'enzyme';
import { HookApp } from '../HookApp';


describe('se renderiza bien HookApp', () => {
    
    test('debe hacer match', () => {
        
        const wrapper = shallow( <HookApp /> );

        expect( wrapper ).toMatchSnapshot();

    });
    

});
