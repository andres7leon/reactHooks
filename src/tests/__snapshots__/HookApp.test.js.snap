// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`se renderiza bien HookApp debe hacer match 1`] = `
<div>
  hla mundoooo
</div>
`;
